content/docs/api.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "The Dex API"
+description: ""
+date: 2020-09-30
+draft: true
+toc: true
+weight: 20
+---
+
+Dex provides a [gRPC](http://www.grpc.io/) service for programmatic modification of dex's state.
```

Comment:
Maintainers: Which is the correct capitalization of Dex – lowercase (at all times) or uppercase?

@hackabletype: You don't need to edit all the body content for capitalization in this PR, but please ensure that the titles (and thus menu) are properly capitalized.

Reply:
Personally, I prefer Dex and tried to use it everywhere. Prior to me joining the project, both was used.

Reply:
I've gone through and made sure that in the titles "Dex" is capitalized as per @sagikazarmark's suggestion.